server/testutils/factories/referral.ts

```diff
@@ -8,6 +8,7 @@ export default Factory.define<Referral>(() => ({
   oasysConfirmed: false,
   offeringId: faker.string.uuid(),
   prisonNumber: faker.string.alphanumeric({ length: 7 }),
+  reason: faker.lorem.sentence(),
```

Comment:
I saw the conversation about `status` on another PR, but I wonder whether the factory default should be a freshly created referral, pushing us to be intentional when using factories to create more complete referrals, and avoid randomness with values that have functional implications

`oasysConfirmed` defaults to `false` here, not a random boolean, for example

Unsure what the default value is based on the migration here (I guess `''` or `undefined` once it hits our service): https://github.com/ministryofjustice/hmpps-accredited-programmes-api/blob/main/src/main/resources/db/migration/V11__extend_referral_table.sql

Reply:
As I said in that conversation, I don't think this is all that important, as we should be explicitly setting the values we care about in the tests where they're relevant.

I know you updated this in your other PR @yndajas and I approved that but I think that if we want to represent a new, empty referral, we should use a [trait](https://github.com/thoughtbot/fishery#adding-reusable-builders-traits-to-factories) to represent this, rather than assuming the default factory-built object will represent this early stage referral.

Reply:
Shall we revert the `status` change from my PR and make `oasysConfirmed` a random boolean in that case?

Reply:
I'll add a card to add a trait, we can tackle that as part of that work